.github/action/license

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) Stefan Stölzle, http://stefan.stoelzle.me
```

Comment:
@awcrosby I'm not sure how to correctly handle licensing here. Should we check this with Legal?

cc @stoe

Reply:
@webknjaz maybe so, maybe @gundalow could add thoughts

Reply:
I’m happy to remove my name from that original license, but would appreciate some credits and maybe a similar license for it going forward.

Reply:
@stoe yeah, that's what I was thinking. I thought that maybe we don't need two licenses in the project because it's confusing. OTOH I've already put your name in the README in this dir. We'll make sure to keep the mention as we add info to docs or the main readme. I've even used git subtree split in order to properly extract your commits and add them here so that your history would be preserved :)

Reply:
@webknjaz also with the root license being the same as this, I agree one license is less confusing, and we can maintain the mention in this dir's README.